src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -362,14 +361,13 @@ private void initializeBuscript() {
     }
 
     private void initializeDestinationFactory() {
-        this.destFactory = new DestinationFactory(this);
-        this.destFactory.registerDestinationType(WorldDestination.class, "");
-        this.destFactory.registerDestinationType(WorldDestination.class, "w");
-        this.destFactory.registerDestinationType(ExactDestination.class, "e");
-        this.destFactory.registerDestinationType(PlayerDestination.class, "pl");
-        this.destFactory.registerDestinationType(CannonDestination.class, "ca");
-        this.destFactory.registerDestinationType(BedDestination.class, "b");
-        this.destFactory.registerDestinationType(AnchorDestination.class, "a");
+        this.destinationsManager = new DestinationsManager(this);
```

Comment:
Nitpick: `XManager` is a super generic name without a lot of actual meaning. I would love to avoid names like this in general and try to shoot for names that are more descriptive of their actual purpose.

Reply:
Further rationale: Generic names very easily lead to seeming like a catch all for any behavior related to `X` (destinations in this case). Overtime they fill up with functionality that is not related to their original purpose.

Reply:
hmm good point, `DestinationProvider` prob better describe what the class does i think